tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -97,6 +97,21 @@ describe('Home - Game List', () => {
         .contains('Game made by other player');
     });
 
+    it('Displays "Empty" when there are no players', () => {
+      const players = [];
+      cy.createGamePlayer({ gameName: 'Game with no players', isRanked: false, players });
```

Comment:
`cy.createGamePlayer()` does not accept a `players` argument. It makes a single api call to create the game, at which point there are never any players in it. So we don't need this third argument for this test.
```suggestion
      cy.createGamePlayer({ gameName: 'Game with no players', isRanked: false });
```